CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contribution Guidelines
+
+Glad to see you here! Thank you for your interest in contributing to jsonresume/resume-schema. This guide details how to contribute to jsonresume/resume-schema in a way that is efficient for everyone.
+
+## Code of conduct
+
+Please note that this project is released with a Contributor [Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms. Please report unacceptable behavior to [INSERT EMAIL ADDRESS]. The [Contributor Covenant](https://github.com/Bantik/contributor_covenant) was orginally made by [Coraline Ada Ehmke (Bantik)](https://github.com/Bantik).
+
+## Creating an issue
+
+* Search if the [issue](https://github.com/jsonresume/resume-schema/issues) exist.
+* If the issue exists, then contribute by comments, or you can create one.
+* Try to be as descriptive, concise and specific as possible.
+* Titles must be able to stand on their own without requiring readers to click on the issue.
+* Submit your issue.
+
+## Creating a pull request
+
+* Check if there is an issue related to your feature. If there is not, create one.
+* Search for an [pull request](https://github.com/jsonresume/resume-schema/pulls) that relates to your submission. You don't want to duplicate effort.
+* [Fork the project](https://help.github.com/articles/fork-a-repo/).
+* Create a [pull request](https://help.github.com/articles/creating-a-pull-request/).
+* Submit your pull request.
```

Comment:
* The pull request should be self contained and provide only changes corresponding to the issue/PR title.
(Or something like that)